src/server/location.rs

```diff
@@ -302,9 +305,19 @@ impl Location {
             Ok(location) => {
                 if let Some(names) = location.city.and_then(|c| c.names) {
                     result.city = get_preferred_language_element(&preferred_languages, &names)
+                } else {
+                    metrics.incr_with_tags(
+                        "location.unknown.city",
+                        Some(&Tags::from_extra(vec![("ip", ip_addr.to_string())])),
```

Comment:
extras are only sent to sentry, so all these one offs in this module are unnecessary for metrics calls (I guess that applies to the new `from_extra` method too)

Reply:
Yeah, I debated adding these as proper tags, but I figured we'd hit cardinality issues pretty fast. I'll remove the tag from this for now. 

As for keeping the `from::extra`, I can see that potentially being useful in the future.